settings.gradle

```diff
@@ -1,3 +1,7 @@
+plugins {
+    id 'com.gradle.enterprise' version '3.1.1'
```

Comment:
Could you move the configuration of the build scan plugin also to `settings.gradle`? I think the other usage of `ciBuild` (`ignoreFailures = ciBuild`) can be removed from `build.gradle`.

Reply:
I think `ignoreFailures = ciBuild` should be kept, otherwise on failure the test results would not be recorded - see [buildPluginWithGradle](https://github.com/jenkins-infra/pipeline-library/blob/1.4.1/vars/buildPluginWithGradle.groovy#L66). There is no central configuration provided by the infrastructure library as it is available for Maven. We already had a discussion at https://github.com/jenkins-infra/pipeline-library/pull/88#issuecomment-521341959.

Reply:
I was hoping that the test results are captured even if one of the builds fails... `ignoreFailures = false` is bad, because the build cache would then cache the result of a test, even if there are test failures (since the test task succeeded). That is why I would prefer the test task to fail if there are test failures.

Since we are now not using the build cache on CI, it is safe to keep the `ignoreFailures = false` flag.